oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticDocument.java

```diff
@@ -148,4 +166,11 @@ public String build() {
     public String toString() {
         return build();
     }
+
+    public static Collection<Field> newSimilarityFields(String name, Blob value) throws IOException {
```

Comment:
I don't think this method is used. Can we remove it with all the import references on Lucene?